### ISSUE TITLE
AttributeError in access_permissions.py after start server

### ISSUE THREAD

**Issue by emanuelschuetze** [OpenSlides/OpenSlides#2908]
If you start OpenSlides with a fresh db (= new session) and there is still an old session in browser tab open the server console shows the following **AttributeError**:

```
2017-01-25 09:56:48,043 - ERROR - worker - Error processing message with consumer openslides.utils.autoupdate.ws_add_site:
Traceback (most recent call last):
  File "/home/emanuel/projects/openslides/src/OpenSlides/.venv/lib/python3.5/site-packages/channels/worker.py", line 119, in run
    consumer(message, **kwargs)
  File "/home/emanuel/projects/openslides/src/OpenSlides/.venv/lib/python3.5/site-packages/channels/sessions.py", line 180, in inner
    result = func(message, *args, **kwargs)
  File "/home/emanuel/projects/openslides/src/OpenSlides/.venv/lib/python3.5/site-packages/channels/auth.py", line 73, in inner
    return func(message, *args, **kwargs)
  File "/home/emanuel/projects/openslides/src/OpenSlides/.venv/lib/python3.5/site-packages/channels/sessions.py", line 64, in inner
    return func(message, *args, **kwargs)
  File "/home/emanuel/projects/openslides/src/OpenSlides/.venv/lib/python3.5/site-packages/channels/auth.py", line 89, in inner
    return func(message, *args, **kwargs)
  File "/home/emanuel/projects/openslides/src/OpenSlides/openslides/utils/autoupdate.py", line 43, in ws_add_site
    output.extend(collection.as_autoupdate_for_user(message.user.id))
  File "/home/emanuel/projects/openslides/src/OpenSlides/openslides/utils/collection.py", line 373, in as_autoupdate_for_user
    content = collection_element.as_autoupdate_for_user(user)
  File "/home/emanuel/projects/openslides/src/OpenSlides/openslides/utils/collection.py", line 126, in as_autoupdate_for_user
    user)
  File "/home/emanuel/projects/openslides/src/OpenSlides/openslides/utils/collection.py", line 108, in as_autoupdate
    *args)
  File "/home/emanuel/projects/openslides/src/OpenSlides/openslides/users/access_permissions.py", line 45, in get_restricted_data
    elif user.pk == full_data.get('id'):
AttributeError: 'NoneType' object has no attribute 'pk'

```

**Comment by normanjaeckel** [comment]
Should be solved with #2909. Test this after merge.

**Comment by emanuelschuetze** [comment]
Confirmed. Fixed in current master.
